WORKSPACE

```diff
@@ -10,7 +10,7 @@ envoy_api_dependencies()
 
 load("//bazel:repositories.bzl", "envoy_dependencies")
 
-envoy_dependencies()
+envoy_dependencies(developer_tools = True)
```

Comment:
this is already controlled by the existence of `LLVM_CONFIG`, do we still need `developer_tools ` here?